R/utils.R

```diff
@@ -19,60 +23,29 @@
 #' )
 #' assay(se)
 #' assay(relativeAb(se))
+#' 
+#' 
+#' utility function for relativeAb to return counts per million
+#'
+#' @description
+#' This function calculates the relative abundance of each feature in a dataset,
+#' expressed as counts per million (CPM) 
+#' 
+#' @param normalize Logical value indicating whether counts should be normalized
```

Comment:
You're adding documentation for an argument `normalize` but as far as I can see, `normalize` isn't currently an argument of the `relativeAb()` function. Am I missing something?

Reply:
Okay, I'll make changes to the documentation